src/OpenIddict.Mvc/OpenIddictController.cs

```diff
@@ -38,7 +38,7 @@ public class OpenIddictController<TUser, TApplication> : Controller where TUser
         protected virtual OpenIddictOptions Options { get; }
 
         [HttpGet, HttpPost]
-        public async Task<IActionResult> Authorize() {
+        public virtual async Task<IActionResult> Authorize() {
```

Comment:
Being able to subclass the default OpenIddict controller and change its logic. I'm fine sealing it if you don't like this change :+1: